examples/standalone_panorama/README.md

```diff
@@ -21,16 +17,24 @@ This is a non zonal deployment. The deployed infrastructure consists of:
   * a Network Security Group to give access to Panorama's public interface
 * a Panorama appliance with a public IP assigned to the management interface
 
+![standalone-panorama](https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/assets/6500664/a2cc53c1-d57e-440a-b0d6-838e2217a20f)
```

Comment:
The image is transparent. I think we should change it to one with a solid background as it's unreadable when using a dark mode in GH
![image](https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/assets/42772730/98a12c97-59eb-412f-ad7d-7bb92edb0c92)

Reply:
Fixed